search_service/api/table.py

```diff
@@ -99,11 +99,11 @@ def get(self, *, field_name: str,
 
         try:
             results = self.proxy.fetch_table_search_results_with_field(
-                query_term=args.get('query_term'),
+                query_term=args.get('query_term', ''),
                 field_name=field_name,
                 field_value=field_value,
                 page_index=args['page_index'],
-                index=args['index']
+                index=args.get('index')
```

Comment:
In what case this will make a difference?

Reply:
if index doesn't exist in args, the previous one will throw exception.